gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -816,25 +849,35 @@ gst_tensor_converter_chain (GstPad * pad, GstObject * parent, GstBuffer * buf)
       }
       break;
     case _NNS_OCTET:
+      if (self->modename != NULL) {
```

Comment:
Q. Is there a case `self->modename == NULL` ?

Reply:
Yes, If the user did not assign mode name. 
mode name is used to register sub-plugin.

Reply:
Thanks for the explanation :)